web/blog/2021-11-21-seed-round.md

```diff
@@ -11,13 +9,15 @@ import useBaseUrl from '@docusaurus/useBaseUrl';
 import InBlogCta from './components/InBlogCta';
 import WaspIntro from './_wasp-intro.md';
 
-<WaspIntro />
-<InBlogCta />
-
 After graduating from Y Combinator's Winter 2021 Batch, we are super excited to announce that Wasp raised $1.5m in our first funding round! The round is led by Lunar Ventures and joined by HV Capital. Also [see it in TechCrunch](https://techcrunch.com/2021/10/04/yc-grads-wasp-land-1-5m-seed-to-help-developers-build-web-apps-faster/).
 
 The best thing about it is that the majority of our investors are either experienced engineers themselves (e.g. ex-Facebook, Twitter and Airbnb) or have a strong focus on investing in deep technology and developer companies. They share the vision we have with Wasp, understand and care about the problem we are solving.
 
+<!--truncate-->
+
+<WaspIntro />
+<InBlogCta />
+
```

Comment:
@matijaSos I've rearranged the text in order to put something meaningful on the card. However, I might have ruined the article's "flow" in the process. Feel free to revise this (everything before the `<!--truncate>` line gets shown on the card).

This is how the card will look if we leave it like this:
![image](https://user-images.githubusercontent.com/22752307/157666833-a1102c8d-3715-4823-878e-a1c8174e42d4.png)

I think the card looks fine, but the full text might be problematic. When you open the full article, the Wasp intro stuff comes after the summary. Unfortunately, _docosaurus_ does not provide a way to show a summary that isn't a prefix of the entire post.

Feel free to revise any of the other summaries as well.

Reply:
I see, makes sense! It's not a huge problem, if we made CTA part a bit more contrasted it would make things even clearer. But there is a separate issue for that, so we don't have to handle it here: https://github.com/wasp-lang/wasp/issues/462

On the other hand, I was just checking out Supabase's blog and saw they were using `description` tag in blog header, which then appears as a summary text:

![Screenshot 2022-03-14 at 14 45 52](https://user-images.githubusercontent.com/1536649/158184883-36c9d6d0-5cd4-46c5-ae27-d038bd49a40a.png)

![Screenshot 2022-03-14 at 14 46 09](https://user-images.githubusercontent.com/1536649/158184937-759ca346-536a-42e5-9f0d-cc42695d663d.png)

Here is the raw file: https://raw.githubusercontent.com/supabase/supabase/ac72bfd5f45304279df1bf7f47cd213452a6c2e4/www/_blog/2021-12-03-launch-week-three-friday-five-more-things.mdx

Is that something we could use?

Reply:
I see in Docusaurus docs they mention `description` property here (https://docusaurus.io/docs/blog#adding-posts), but couldn't find the "official" docs for it, so not sure how it works.

Reply:
I'll try it out. Assumed they didn't support it since I found an open issue asking for the feature. Description might end up being something else, but perhaps we get lucky. Thanks!